.github/workflows/win-wheels.yaml

```diff
@@ -43,15 +44,15 @@ jobs:
           cat setup.cfg
 
       - name: Cache vcpkg
-        uses: actions/cache@v1
+        uses: actions/cache@v3
         id: vcpkgcache
         with:
           path: C:/vcpkg/installed
-          key: ${{ runner.os }}-vcpkg-${{ hashFiles('.github/workflows/*') }}
+          key: ${{ runner.os }}-dependencies-${{ hashFiles('.github/workflows/*', 'vcpkg.json') }}
```

Comment:
@snowman2 while you're saving my bacon, do you see anything obviously wrong with the above? I'm having the experience of not seeing the cache miss when I change vcpkg.json.

Reply:
At a glance, it looks fine. I guess we'll just have to see if it works 🤞.

In case it is helpful, pyproj uses the expected PROJ version in the key to update the cache: https://github.com/pyproj4/pyproj/blob/bc7805ccc813b40ab28099ef305821300f149aad/.github/workflows/release.yaml#L93.